### DISCUSSION TITLE
how to use and download of profiles

### DISCUSSION THREAD

**Discussion by ordiordi** [DIGITALCRIMINALS/OnlyFans#104] [Q&A]
hello
i don't know how to use the scrapper , i have no idea if i need to use the link or the username, and of course i don't know where to put it
so it would be much appreciated if someone could explain it for me

**Comment by DIGITALCRIMINAL** [comment]
Just follow the instructions

**Reply by ghost** [reply]
I followed the instructions. When I open "start_ofd.py", I select OnlyFans, my profile,, all suscriptions, and it sais this: 
![image](https://user-images.githubusercontent.com/97457722/148986398-cc915e67-1634-4167-93b8-c8b28af4272d.png)

**Reply by m4rc1nt0sch** [reply]
You obviously don't follow any creators on OF with the account credentials you signed in, because the Sign-In process worked successfully (it tries to look for paid content or subscriptions). It simply does not work if you're not subscribed to a content creator, and it is intended to not work without having a valid subscription.
